src/liquid/components/ld-bg-cells/ld-bg-cells.tsx

```diff
@@ -34,12 +35,47 @@ export type CellType =
   shadow: true,
 })
 export class LdBgCells {
+  @Element() element: HTMLElement
+
   /** Cells pattern */
   @Prop() type: CellType = 'hexagon'
 
   /** Use 3 color layers */
   @Prop() threeLayers = false
 
+  /** Animate the pattern */
+  @Prop() animated = false
+
+  @Watch('type')
+  private async loadPatternPathData(): Promise<void> {
+    if ((!Build.isBrowser && !Build.isTesting) || !this.type) {
+      return
+    }
+
+    const patternString = await fetchPattern(this.type)
+    const div = document.createElement('div')
+    div.innerHTML = patternString
+
+    const secondaryDiv = document.createElement('div')
+    secondaryDiv.innerHTML = patternString
+
+    Array.from(this.element.shadowRoot.children).forEach((child) => {
+      if (child.classList.contains('ld-bg-cells__layer')) {
+        const svg = div.firstChild as SVGElement
+        child.innerHTML = svg.innerHTML
+      }
+
+      if (child.classList.contains('ld-bg-cells__secondary-layer')) {
+        const svg = secondaryDiv.firstChild as SVGElement
+        child.innerHTML = svg.innerHTML
+      }
+    })
+  }
+
+  async componentWillLoad(): Promise<void> {
+    this.loadPatternPathData()
+  }
```

Comment:
```suggestion
 componentWillLoad() {
    this.loadPatternPathData()
  }
```

You are not awaiting anything here.
You'll have to update the snapshots again.

Reply:
Same here, copy & paste from ld-icon. Will also adapt.